modules/detection/lidar_driver.py

```diff
@@ -0,0 +1,270 @@
+"""
+Lightware SF45/B Driver
+Lidar documentation: https://support.lightware.co.za/sf45b/#/commands
+NEEDS TO BE TESTED
+"""
+
+import time
+import struct
+import serial
+
+
+class LidarDriver:
+    """
+    Wrapper for lidar
+    """
+
+    def __init__(self, port_name, baudrate, timeout):
+        """
+        port_name: port that the lidar is connected to
+        baudrate: baudrate of the lidar port
+        timeout: timeout for connecting to serial port
+        """
+        self.serial_port = serial.Serial(port_name, baudrate, timeout=timeout)
+        self._packet_parse_state = 0
+        self._packet_payload_size = 0
+        self._packet_size = 0
+        self._packet_data = []
+
+    def _create_crc(self, data):
```

Comment:
use double leading underscores for private methods (this does something called mangling, which prevents naming conflicts with other classes)